src/odemis/gui/cont/project.py

```diff
@@ -508,27 +512,21 @@ def _on_coordinates(self, _=None):
         Whenever the list of active scintillators changes and or a ROC is selected/deselected, the
         buttons are updated/enabled/disabled accordingly.
         """
-        rocs = self.calibration_regions.value
         active_scintillators = self._tab_data.main.active_scintillators.value
 
         for num, b in self.panel.buttons.items():
-            roc = rocs[num]
-
-            # scintillator selected, but undefined ROC
-            if num in active_scintillators and roc.coordinates.value == acqstream.UNDEFINED_ROI:
-                b.Enable(True)
-                b.SetLabel("?")
-                b.SetForegroundColour(odemis.gui.FG_COLOUR_RADIO_INACTIVE)
-            # scintillator selected, defined roc
-            elif num in active_scintillators and roc.coordinates.value != acqstream.UNDEFINED_ROI:
-                b.Enable(True)
-                b.SetLabel("OK")
-                if self._calib_prefix == "calib_2":
-                    b.SetForegroundColour(FG_COLOUR_WARNING)
-                else:
-                    b.SetForegroundColour(wx.GREEN)  # default to green
-            # scintillator unselected
-            else:
+            if num in active_scintillators:
+                b.Enable(True)  # always enable the button when the scintillator is active
+                if self.roc_ctrls[num].is_selected:
+                    b.SetLabel("OK")
+                    if self._calib_prefix == "calib_2":
+                        b.SetForegroundColour(FG_COLOUR_WARNING)
+                    else:
+                        b.SetForegroundColour(wx.GREEN)  # default to green
+                elif not self.roc_ctrls[num].is_selected:
```

Comment:
is_selected is a boolean, so it should be that simple:
```suggestion
                else:
```